### PR TITLE
Changed yq path

### PR DESCRIPTION
**Pull request by smark88**
additionally added paths for bumper so semvar doesn't always bump commit to develop. 

path change PR: https://github.com/broadinstitute/datarepo-helm-definitions/pull/123

### CODE REVIEW

**Review by smark88 — COMMENTED** [review]
Need group consensus on what paths should trigger this.

@fboulnois @snf2ye

File: .github/workflows/gradle-build-pr.yml [hunk 1]
```diff
@@ -9,6 +9,11 @@ on:
   pull_request:
     branches:
       - develop
+    paths:
```
Comment (smark88) [inline .github/workflows/gradle-build-pr.yml:12]: Need group consensus on what paths should trigger this.

@fboulnois @snf2ye

File: .github/workflows/dev-image-update.yaml [hunk 2]
```diff
@@ -9,6 +9,11 @@ on:
   push:
     branches:
       - develop
+    paths:
```
Comment (smark88) [inline .github/workflows/dev-image-update.yaml:12]: Need group consensus on what paths should trigger this.

@fboulnois @snf2ye

**Review by fboulnois — APPROVED** [review]
This lgtm once we have group consensus on paths.

**Review by snf2ye — APPROVED** [review]

**Review by nmalfroy — APPROVED** [review]

**Pull request merged**